appveyor.yml

```diff
@@ -131,10 +135,9 @@ for:
 
       # if we are testing a pull request, building only a version will be sufficient, so stop this script
       - |-
-        if [ -n "${APPVEYOR_PULL_REQUEST_NUMBER}" ]; then
+        if [ "${APPVEYOR_PULL_REQUEST_NUMBER}" ]; then
```

Comment:
Is there a particular reason why -n was removed?

Reply:
no